gauth.go

```diff
@@ -17,13 +19,16 @@ import (
 )
 
 func main() {
+	useCSV := flag.Bool("csv", false, "Output CSV for easy machine processing")
```

Comment:
A minor cosmetic point:

This is functionally correct, but it's fairly conventional to put flag definitions at global scope (done during initialization) so they are visible to the whole main package:

```go
var (
  useCSV = flag.Bool("csv", false, "Output CSV for easy machine processing")
)

func main() {
  flag.Parse()
  // …
}
```

There's only one right now, and it's only used inside `main` at the moment, but based on some of the other open issues I predict there will be more flags coming along—and it'd be nice if they could be accessed from other helpers in the main package.

YMMV.

Reply:
Oh thanks for the reminder, haven't written go or touched that code in a long time, 💯 must fix.

Reply:
Done.